src/components/config-pages/global-message.tsx

```diff
@@ -24,38 +22,30 @@ const GlobalMessage = ({
   });
 
   return (
-    <div className={wrapperClasses + " py-10"}>
-      <div className="centered flex flex-col lg:flex-row gap-10">
-        <div className="flex items-center leading-none shrink-0">
-          <MessageIcon messageType={suGlobalMsgType}/>
-          {suGlobalMsgLabel}:
-        </div>
-        <div className="[&_a]:text-white [&_a.btn]:bg-transparent [&_a.btn]:border-2 [&_a.btn]:border-white">
-          {suGlobalMsgHeader && <H2>{suGlobalMsgHeader}</H2>}
-
-          <Wysiwyg html={suGlobalMsgMessage?.processed}/>
+    <div className='bg-fog-light'>
```

Comment:
Do we want this? it puts a background behind the alert that is different than the background behind the menu/lockup area. If so, use double quotes.